content/apps/create-app/callback-php.md

```diff
@@ -24,25 +27,28 @@ if (empty($pimUrl)) {
     exit('No PIM url in session');
 }
 
+// Generate code for token request
 $codeIdentifier = bin2hex(random_bytes(30));
-$codeChallenge = hash('sha256', $codeIdentifier . OAUTH_CLIENT_SECRET);
+$codeChallenge = hash('sha256', $codeIdentifier . $oauthClientSecret);
 
-$accessTokenUrl = $pimUrl . '/connect/apps/v1/oauth2/token';
+// Build form data to post
 $accessTokenRequestPayload = [
-    'client_id' => OAUTH_CLIENT_ID,
+    'client_id' => $oauthClientId,
     'code_identifier' => $codeIdentifier,
     'code_challenge' => $codeChallenge,
     'code' => $authorizationCode,
     'grant_type' => 'authorization_code',
 ];
 
-// Do a POST request on the token endpoint
-$ch = curl_init();
-curl_setopt($ch, CURLOPT_URL, $accessTokenUrl);
-curl_setopt($ch, CURLOPT_POST, true);
-curl_setopt($ch, CURLOPT_POSTFIELDS, $accessTokenRequestPayload);
-curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-$response = json_decode(curl_exec($ch), true);
+// If you haven't set your client yet, please install Guzzle by following the <a href="https://docs.guzzlephp.org/en/stable/overview.html#installation" target="_blank" rel="noopener noreferrer">official documentation</a>
```

Comment:
```suggestion
// If you haven't set your client yet, please install Guzzle by following the official documentation: https://docs.guzzlephp.org/en/stable/overview.html#installation
```
<a> is not interpreted in code markdown :(

Reply:
may require to jump a line for visibility

```php
// If you haven't set your client yet, please install Guzzle by following the official documentation:
// https://docs.guzzlephp.org/en/stable/overview.html#installation
```